x-pack/plugins/security_solution/public/detections/components/rules/rule_preview/preview_histogram.tsx

```diff
@@ -214,7 +213,12 @@ export const PreviewHistogram = ({
             dataProviders,
             deletedEventIds,
             disabledCellActions: FIELDS_WITHOUT_CELL_ACTIONS,
-            end: alertsEndDate,
+            // Fix for https://github.com/elastic/kibana/issues/135511, until we start writing proper
+            // simulated @timestamp values to the preview alerts. The preview alerts will have @timestamp values
+            // close to the server's `now` time, but the client clock could be out of sync with the server. So we
+            // avoid computing static dates for this time range filter and instead pass in a small relative time window.
+            end: 'now+5m',
+            start: 'now-5m',
```

Comment:
Not sure if it fixes https://github.com/elastic/kibana/issues/135511 and, well, I don't even know what `start` and `end` parameters mean here, but yeah, looks like we need some deterministic implementation for generating `@timestamp` values in rule preview and then filtering by `@timestamp`.